test/transforms.jl

```diff
@@ -598,6 +598,45 @@
     @test isequalmissing(n.e, [missing, 5, 6, 5])
     @test isequalmissing(n.f, [4, missing, 4, 5])
 
+    # column eltype
+    ttypes = Tables.schema(t).types
+
+    T = DropMissing()
+    n, c = apply(T, t)
+    ntypes = Tables.schema(n).types
+    @test ntypes[1] == Int
+    @test ntypes[2] == Int
+    @test ntypes[3] == Int
+    @test ntypes[4] == Int
+    @test ntypes[5] == Int
+    @test ntypes[6] == Int
+    tₒ = revert(T, n, c)
```

Comment:
move this revert call to after the apply call

Reply:
Made in [1514725](https://github.com/JuliaML/TableTransforms.jl/pull/35/commits/151472520bcb0c3d5222738bfa33dce003172cf7)